fatips/1.md

```diff
@@ -1,226 +1,171 @@
-| FATIP | Title                       | Status | Category       | Author                             | Requires | Created    |
-| ----- | --------------------------- | ------ | -------------- | ---------------------------------- | -------- | ---------- |
-| 1     | Non-Fungible Token Standard | Draft  | Token Standard | Devon Katz \<<devonk@dbgrow.com>\> | -        | 2018-07-27 |
+| FATIP | Title                       | Status   | Category       | Author                             | Created    |
+| ----- | --------------------------- | -------- | -------------- | ---------------------------------- | ---------- |
+| 1     | Non-Fungible Token Standard | Accepted | Token Standard | Devon Katz \<<devonk@dbgrow.com>\> | 2018-07-27 |
 
 
 
 # Overview
 
-This document describes the functionality, datastructures, validation rules of
-the **FAT-1** token standard: A non-fungible data only token standard built on
-Factom chains and entries. It's functionality is most alike the ERC-721 token
-standard of Ethereum.
+This document describes the **FAT-1** token standard: A non-fungible (NF) token standard. This this standard allows for token systems supporting unique, indivisible tokens issued in whole units. It is most equivalent to Ethereum's ERC-721 token standard.
+
 
 
 # Motivation
 
-Extend upon the FAT-0 standard by offering non-fungible token functionality for
-unique, indivisible, and traceable assets.
+Offer an efficient non-fungible token system, allowing attachment of arbitray metadata to issued tokens. Allow unique, indivisible, and traceable assets on top of FAT.
+
 
-Applications include tokenized equity, commodities, financial instruments, and
-digital goods.
 
+# Specification `inherits` [FATIP-0](0.md)
 
-# Specification
+FATIP-1 draws heavily upon the  [FATIP-0](0.md) fungible token standard for it's data structures and signing.
 
+All core functionality from FATIP-0 is inherited by this standard, unless otherwise specified or overridden. Equivalent section headers will define which functionality is built upon, modified, or replaced.
 
-## Token ID  `extends` [FATIP-100](100.md)
 
 
-## Addresses `extends` [FATIP-0](0.md)
+## Non-Fungible Token IDs
 
+Non-fungible tokens issued on FATIP-1 are uniquely identified by an non-negative integer ID. For example, token ID `5`.
 
-## Token Issuance Chain
+### Token ID Ranges
 
-Unlike FAT-0, the issuance chain can contain multiple entries. FAT-1 allows the
-ongoing issuance of unique tokens over time up to a maximum supply.
+ FATIP-1 has the unique ability to specify sequential ranges of non-fungible token IDs during issuance and transaction. For example, the range `{"min": 0, "max":4410}` represents the range of token IDs `0` to `4410`, inclusive.
```

Comment:
does fatd acceppt 0 @AdamSLevy ? I think I tried few days ago and I had to start at 1.